Dockerfile

```diff
@@ -37,22 +34,22 @@ RUN mkdir -p /src
 ADD https://cloud.gastecnologia.com.br/gas/diagnostico/warsaw_setup_64.deb /src/GBPCEFwr64.deb
 	# Configuring the environment
 RUN mkdir -p /home/${USER} \
-	&& chmod 744 /home/ff/startup.sh \
 	&& groupadd -g ${GUID} -r ${USER} \
 	&& useradd -u ${GUID} -r -g ${USER} -G audio,video ${USER} -d /home/${USER} \
 	&& chown -R ${GUID}:${GUID} /home/${USER} \
-	&& echo 'ff ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers \
-	&& echo 'Defaults !requiretty' >> /etc/sudoers \
-	&& echo root:wscef | chpasswd \
 	# Cleanup
 	&& apt autoremove -y \
 	&& apt clean
 
-# Run Firefox as non privileged user
-USER ff
+RUN apt -y install /src/GBPCEFwr64.deb || :
+
+COPY root.sh /usr/local/bin/
+COPY startup.sh /usr/local/bin/
+RUN chmod 700 /usr/local/bin/root.sh \
+    && chmod 755 /usr/local/bin/startup.sh
 
-# Add volume for recipes PDFs
+# Add volume for receipts PDFs
 VOLUME "/home/ff/Downloads"
 
 # Autorun Firefox
-ENTRYPOINT /home/ff/startup.sh
+ENTRYPOINT /usr/local/bin/root.sh
```

Comment:
Qual é a necessidade de um novo init e executando como usuário root em vez de usar o sudo? Fora também retirar o init do diretório home do usuário sem privilegio até então

Reply:
Troquei o diretório dos script para deixar no $PATH. Me parece mais conveniente, assim ambos scripts podem ficar no mesmo diretório..

Reply:
Ah, o init é para fazer zombie reaping, etc. O processo root apenas executa o warsaw root, e já reverte para usuário não-privilegiado. A configuração anterior sudoers deixa aberto para escalação de privilégios.

Reply:
Ok. Penso eu se restringir mais o sudo resolveria a questão?

Reply:
Entendo que fiz algumas mudanças aparentemente desnecessárias. O caso do sudo sendo um programa suid enxergo que neste ponto a solução que proponho é superior, toda vida.

Reply:
Foi o que me disseram... no começo do projeto usava o su... depois sugeriram o sudo para evitar de intervenção durante o bootstraping da solução

Reply:
Achei este artigo útil, sobre privilégios: http://jdebp.eu./FGA/dont-abuse-su-for-dropping-privileges.html